llvm/test/Bitcode/ptrtoaddr.ll

```diff
@@ -0,0 +1,14 @@
+; RUN: llvm-as < %s | llvm-dis | llvm-as | llvm-dis | FileCheck %s
```

Comment:
Does it really need to do 2 round trips? I don't think other bitcode tests do this?

Reply:
I copied this from another test so assumed there was a reason for it but not sure what that would be.